api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,126 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user) who share access to resources in Microsoft services or in your app. Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs. All operations in Microsoft Graph on groups require administrator consent.
+
+**NOTE**
+Groups are only supported for work or school accounts, and not supported for Microsoft personal accounts.
+
+There are 2 general types of groups, distinguished by their use cases and specific properties' values on each resource. 
+
+| Type              | Use case | groupType | mail-enabled | security-enabled | 
+|-------------------|----------|-----------|--------------|------------------|
+| Office 365 groups | Facilitating user collaboration with shared Microsoft online resources. | ["Unified"] | true | false | 
+| Security groups | Controlling user access to in-app resources. | [] | false | true |

```

Comment:
Actually Jimaco - that isn't true.  SGs are most commonly/primarily used to restrict/control access to a variety of different services, including LoB enterprise apps, and ISV apps.  We should not call these Azure AD SGs.  Security groups is good.
Vivian - I like the way you've laid this out.  In the first row, I give examples for shared Microsoft online resources (like calendar, conversation, document library etc).

I think (unfortunately) we probably need to mention Mail-enabled security groups and Distribution Groups.